content/24.vaccines-novel.md

```diff
@@ -492,75 +499,28 @@ A 50 &mu;g dose of mRNA-1273.214 was administered to 437 participants.
 One month later, the neutralizing geometric mean titer ratio was assessed against several variants of SARS-CoV-2, including Omicron.
 The immune response was higher against all variants assessed, including Omicron, than for boosting with the original formulation (mRNA-1273).
 Another formulation, mRNA-1273.211, developed based on the Beta variant, has been associated with durable protection as long as six months after dosing.
-These data are available through a press release and have not yet been published.
-Therefore, additional results may provide better insight into the VE of these formulations.
+The associated publications suggest that this novel formulation offers significant protection against Omicron and other VOC [@doi:10.1038/s41591-021-01527-y; @doi:10.1056/NEJMc2119912]. 
+In August 2022, Pfizer also announced successful development of a new formulation effective against Omicron [@doi:https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-announce-omicron-adapted-covid-19].
+
+Modularity has been proposed as one of the advantages to developing DNA and mRNA vaccines.
+This design would allow for faster adaptation to viral evolution.
+However, in the arms race against SARS-CoV-2, the vaccines are still lagging behind the virus.
+This disadvantage may change as regulators become more familiar with these vaccines and as a critical mass of data is accumulated.
 Given the apparent need for boosters, interest has also emerged in whether updated formulations of SARS-CoV-2 vaccines can be administered along with annual flu vaccines to improve immunity to novel variants.
-<!-- What is our conclusion about this (or will that go in the Conclusion?). The potential of mRNA vaccines was rapid reformulation, but that hasn't really happened in practice (yet). -->
-
-### Effect of Vaccines on Community Spread
-<!-- I suggest moving this entire section to the appendix. It is well written but long, not directly related to the rest of the sections, and somewhat inconclusive. We could keep the last sentence as part of the conclusions ("Evidence suggests that public health measures ...") and refer to the appendix to support that statement. -->
```

Comment:
I moved it to pathogenesis, since it seems like they are hoping we will submit updates, and this seems like worthwhile ground to cover if we update that one.